src/scandeval/data_models.py

```diff
@@ -206,6 +206,7 @@ class BenchmarkConfigParams(pydantic.BaseModel):
     api_version: str | None
     debug: bool
     run_with_cli: bool
+    only_allow_safetensors: bool = False
```

Comment:
Having a default value here is error-prone - we do that elsewhere to avoid silent errors (which is currently what is happening now).

```suggestion
    only_allow_safetensors: bool
```